packages/kvstore/lib/index.ts

```diff
@@ -1,17 +1,42 @@
+import type { RedisClientType } from 'redis';
+import { createClient } from 'redis';
 import { InMemoryKVStore } from './InMemoryStore.js';
 import { RedisKVStore } from './RedisStore.js';
 
 export { InMemoryKVStore } from './InMemoryStore.js';
 export { RedisKVStore } from './RedisStore.js';
 export type { KVStore } from './KVStore.js';
+export { FeatureFlags } from './FeatureFlags.js';
+export type { RedisClientType } from 'redis';
 
+export const redisURL = process.env['NANGO_REDIS_URL'];
+let kvStore: RedisKVStore | InMemoryKVStore | undefined;
 export async function createKVStore() {
```

Comment:
I would rename `createKVStore` to something like `getKVStore` or similar since it now can return a memoized instance and doesn't always create a new instance